### PR TITLE
test: add friendly tests

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

File: test/getStats.spec.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+'use strict'
```
Comment (douglasduteil) [inline test/getStats.spec.js]: I could add the ";" later if you like them :)

**Comment by ThomasCrvsr** [comment]
Nice ! :)

File: test/getStats.spec.js (hunk 1 above)
Comment (ThomasCrvsr) [inline test/getStats.spec.js]: No need to ^^

Comment (douglasduteil) [inline test/getStats.spec.js]: I went 
[js-standard-style](https://github.com/feross/standard)

**Pull request merged**